spec/spec_helper.rb

```diff
@@ -11,6 +11,9 @@
 $LOAD_PATH.unshift File.expand_path('../..', __FILE__)
 
 RSpec.configure do |config|
+  require_relative './support/global_helpers.rb'
```

Comment:
i saw that, still it doesn't work. doesn't look like it looks into `spec/support`. plus, `require_relative` is O(1) which is why i prefer it until it's very ugly (file path is very long)